Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -201,8 +201,11 @@ private func rewardTitle(project: Project, reward: Reward) -> String {
 private func pillStrings(project: Project, reward: Reward) -> [String] {
   var pillStrings: [String] = []
 
-  if project.state == .live, let endsAt = reward.endsAt, endsAt > 0,
-    endsAt >= AppEnvironment.current.dateType.init().timeIntervalSince1970 {
+  if project.state == .live,
+    let endsAt = reward.endsAt,
+    endsAt > 0,
+    endsAt >= AppEnvironment.current.dateType.init().timeIntervalSince1970,
+    reward.limit == nil || reward.remaining ?? 0 > 0 {
```

Comment:
Can we extract this last piece to a var?

`let isUnlimitedOrAvailable = reward.limit == nil || reward.remaining ?? 0 > 0`

Come to think of it, you probably don't need the first part of that statement because if it's an "unlimited" reward then `reward.remaining` will be `nil` so you'll still get the same outcome. So:

`let isAvailable = reward.remaining ?? 0 > 0` would suffice?

You're the expert on this though 😛

Reply:
😄 I extracted it to a variable for readability.

Actually, we do need the OR, coalescing the nil `remaining` value would assume it's a limited reward. We're saying - return the string if this reward is unlimited OR if it has more than 0 remaining. If the left side is `nil` the second is ignored.

Tidied it up a little though.